Add missing boilerplate header to the files in the packaging/flavorgen directory

/kind bug

**What steps did you take and what happened:**
`make verify` fails with the below errors - 
```
./hack/verify-boilerplate.sh
Boilerplate header is wrong for: /go/src/sigs.k8s.io/cluster-api-provider-vsphere/packaging/flavorgen/flavors/env/envsubts_consts.go
Boilerplate header is wrong for: /go/src/sigs.k8s.io/cluster-api-provider-vsphere/packaging/flavorgen/flavors/crs/csi.go
Boilerplate header is wrong for: /go/src/sigs.k8s.io/cluster-api-provider-vsphere/packaging/flavorgen/flavors/crs/util.go
Boilerplate header is wrong for: /go/src/sigs.k8s.io/cluster-api-provider-vsphere/packaging/flavorgen/flavors/crs/cpi.go
make[1]: *** [verify-boilerplate] Error 1
```


**What did you expect to happen:**
Make verify should successfully execute


/assign

--- a/packaging/flavorgen/flavors/crs/cpi.go
+++ b/packaging/flavorgen/flavors/crs/cpi.go
@@ -1,3 +1,19 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
 package crs
 
 import (

--- a/packaging/flavorgen/flavors/crs/csi.go
+++ b/packaging/flavorgen/flavors/crs/csi.go
@@ -1,3 +1,19 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
 package crs
 
 import (

--- a/packaging/flavorgen/flavors/crs/util.go
+++ b/packaging/flavorgen/flavors/crs/util.go
@@ -1,3 +1,19 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
 package crs
 
 import (

--- a/packaging/flavorgen/flavors/env/envsubts_consts.go
+++ b/packaging/flavorgen/flavors/env/envsubts_consts.go
@@ -1,3 +1,19 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
 package env
 
 const (

